packages/office-ui-fabric-react/src/components/ComboBox/ComboBoxPage.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import {
+  ExampleCard,
+  IComponentDemoPageProps,
+  ComponentPage,
+  PropertiesTableSet
+} from '@uifabric/example-app-base';
+import { ComboBoxBasicExample } from './examples/ComboBox.Basic.Example';
+
+const ComboBoxBasicExampleCode = require('!raw-loader!office-ui-fabric-react/src/components/ComboBox/examples/ComboBox.Basic.Example.tsx') as string;
+
+export class ComboBoxPage extends React.Component<IComponentDemoPageProps, {}> {
+  public render() {
+    return (
+      <ComponentPage
+        title='ComboBox'
+        componentName='ComboBoxExample'
+        exampleCards={
+          <ExampleCard title='ComboBox' code={ ComboBoxBasicExampleCode }>
+            <ComboBoxBasicExample />
+          </ExampleCard>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={ [
+              require<string>('!raw-loader!office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts')
+            ] }
+          />
+        }
+        overview={
+          <div>
+            <p>
+              A ComboBox is a list in which the selected item is always visible, and the others are visible on demand by clicking a drop-down button or by typing in the input (unless allowFreeform and autoComplete are both false). They are used to simplify the design and make a choice within the UI. When closed, only the selected item is visible. When users click the drop-down button, all the options become visible. To change the value, users open the list and click another value or use the arrow keys (up and down) to select a new value. When collapsed if autoComplete and/or allowFreeform are true, the user can select a new value by typing.

```

Comment:
@jspurlin and I talked about it in person and there are quite a few nuances that don't translate perfectly. If we can render exactly the same thing dropdown renders inside of the CheckBox, that I'm good on deprecating. Until then we would have to have 2 here. Maybe that's ok.